docs/introduction.md

```diff
@@ -0,0 +1,138 @@
+
+Because Hooks are the future-facing way to write your React components, we wrote this introduction using function component examples. Where useful, we also cover class components under a toggle:
+
+<div class="toggler">
+  <ul role="tablist" class="toggle-syntax">
+    <li id="functional" class="button-functional" aria-selected="false" role="tab" tabindex="0" aria-controls="functionaltab" onclick="displayTabs('syntax', 'functional')">
+      Function Component Example
+    </li>
+    <li id="classical" class="button-classical" aria-selected="false" role="tab" tabindex="0" aria-controls="classicaltab" onclick="displayTabs('syntax', 'classical')">
+      Class Component Example
+    </li>
+  </ul>
+</div>
+
+<block class="functional syntax" />
+
+```SnackPlayer name=Hello%20World%20Function%20Component
+import React from 'react';
+import { Text, View } from 'react-native';
+
+export default function YourApp() {
+  return (
+    <View style={{
+        flex: 1,
+        justifyContent: 'center',
+        alignItems: 'center'
+      }}>
+      <Text>Hello, world!</Text>
+    </View>
+  );
+}
+```
+
+<block class="classical syntax" />
+
+```SnackPlayer name=Hello%20World%20Class%20Component
+import React, { Component } from 'react';
+import { Text, View } from 'react-native';
+
+export default class HelloWorldApp extends Component {
+  render() {
+    return (
+      <View style={{
+          flex: 1,
+          justifyContent: "center",
+          alignItems: "center"
+        }}>
+        <Text>Hello, world!</Text>
+      </View>
+    );
+  }
+}
+
+```
+
+<block class="endBlock syntax" />
+
+You can find more examples of class components in [previous versions of this documentation](/react-native/versions).
+
+## Developer Notes
+
+People with many different developer experiences are learning React Native. You may have experience with a range of technologies, from web to Android to iOS and more. We try to write for developers from all backgrounds. Sometimes we provide explanations specifc to one platform or another like so:
+
+<div class="toggler">
+  <span>Developer Notes</span>
+  <span role="tablist" class="toggle-devNotes">
+    <button role="tab" class="button-webNote" onclick="displayTabs('devNotes', 'webNote')">Web</button>
+    <button role="tab" class="button-androidNote" onclick="displayTabs('devNotes', 'androidNote')">Android</button>
+    <button role="tab" class="button-iosNote" onclick="displayTabs('devNotes', 'iosNote')">iOS</button>
+  </span>
+</div>
+
+<block class="webNote devNotes" />
+
+> Web developers may be familiar with this concept.
+
+<block class="androidNote devNotes" />
+
+> Android developers may be familiar with this concept.
+
+<block class="iosNote devNotes" />
+
+> iOS developers may be familiar with this concept.
+
+<block class="endBlock devNotes" />
+
+## Formatting
+
+Menu paths are written in bold and use carats to navigate submenus. Example: **Android Studio > Preferences**
+
+Inline code
```

Comment:
I'm not sure what this is referring to. Was this meant to be in backticks? Or is this a header?

![Screen Shot 2020-01-15 at 8 07 34 PM](https://user-images.githubusercontent.com/810438/72467378-b22a5500-37d2-11ea-9b15-c6011ae14a78.png)

Reply:
What the... that looks like something is broken!